examples/fedprox_example/server.py

```diff
@@ -51,7 +51,7 @@ def main(config: Dict[str, Any], server_address: str) -> None:
     # This function will be used to produce a config that is sent to each client to initialize their own environment
     fit_config_fn = partial(
         fit_config,
-        config["local_epochs"],
+        config.get("local_epochs", config["local_steps"]),
```

Comment:
Note that the `fit_config` still function still packages this value as `"local_epochs": local_epochs` with this change, which isn't quite what we want. A possible way to do this would be 
```
def fit_config(
    local_epochs: Optional[int],
    local_steps: Optional[int],
    batch_size: int,
    n_server_rounds: int,
    reporting_enabled: bool,
    project_name: str,
    group_name: str,
    entity: str,
    current_round: int,
) -> Config:
    config = {"local_epochs": local_epochs} if local_epochs else {"local_steps": local_steps}
    return config.update(
        {
            "batch_size": batch_size,
            "n_server_rounds": n_server_rounds,
            "current_server_round": current_round,
            "reporting_enabled": reporting_enabled,
            "project_name": project_name,
            "group_name": group_name,
            "entity": entity,
        }
    )
```
and
```
    fit_config_fn = partial(
        fit_config,
        config.get("local_epochs", None),
        config.get("local_steps", None),
        config["batch_size"],
        config["n_server_rounds"],
        config["reporting_config"].get("enabled", False),
        # Note that run name is not included, it will be set in the clients
        config["reporting_config"].get("project_name", ""),
        config["reporting_config"].get("group_name", ""),
        config["reporting_config"].get("entity", ""),
    )
```

Reply:
ok! Got it now. It took me a few attempts but I managed to make it work now with a slightly different code. Execution is down to 3 minutes now :)